Wikipedia/Code/InsertMediaSettingsViewController.swift

```diff
@@ -208,8 +218,50 @@ final class InsertMediaSettingsViewController: ViewController {
         tableView.separatorStyle = .none
         tableView.tableHeaderView = imageView
         tableView.tableFooterView = buttonView
+
+        if FeatureFlags.needsImageRecommendations {
```

Comment:
> Make sure it still works as usual.

Editor flow is broken here on Staging - the "Next" button doesn't do anything.

Instead of the feature flag, you could set this as a view controller property instead (`fromImageRecommendations`, or something like that), default it to false and set it from the outside at the when you instantiate this from the image recommendations flow. This will leave the editor flow unchanged in the Staging scheme.